lib/models/builder.js

```diff
@@ -18,7 +18,8 @@ module.exports = Task.extend({
     process.env.EMBER_ENV = process.env.EMBER_ENV || this.environment;
 
     var broccoli = require('broccoli');
-    this.tree    = broccoli.loadBrocfile();
+    var brocFile = broccoli.loadBrocfile();
```

Comment:
We should not use `loadBrocfile` since we are intending something else (this dovetails with my concern over naming the file `Brocfile.js`).